src/webui/api/appcontroller.cpp

```diff
@@ -293,8 +293,10 @@ void AppController::preferencesAction()
 
     // Advanced settings
     // qBitorrent preferences
+#ifdef QBT_USES_LIBTORRENT2
```

Comment:
And you should adjust the description of the option on WebUI, mention it requires libtorrent >= 2.0. You can find examples on other options.